lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -8,35 +8,40 @@ module Fastlane
 
   module Helper
     class VersioningHelper
-      def self.determine_next_version_using_labels(repo_name, github_token, rate_limit_sleep)
+      # rubocop:disable Metrics/CyclomaticComplexity
+      def self.determine_next_version_using_labels(repo_name, github_token, rate_limit_sleep, skip_if_no_public_changes)
         old_version = latest_non_prerelease_version_number
         UI.important("Determining next version after #{old_version}")
 
         commits = Helper::GitHubHelper.get_commits_since_old_version(github_token, old_version, repo_name)
 
         type_of_bump = :patch
-
+        should_skip_release = true
```

Comment:
Hmm I wonder if we should rename this to `has_public_changes = false` and set to true if we find a PR with public changes... I think I find it less confusing and it better represents what this variable contains.